### ISSUE TITLE
game freezes and ryze

### ISSUE THREAD

**Issue by djole353** [berbb/PortAIO#26]
ok so everything works fine but once in 4 or 5 games the game freezes and i have to restart it, is there any solution? also are you going to add addons for reworked ryze

**Comment by yorik100** [comment]
Same problems :/

**Comment by berbb** [comment]
You haven't given me anything that would help me solve it so I'm not sure how I would. The big one is what utilities are you using? And yes, most ported addons for Ryze are already updated.

**Comment by yorik100** [comment]
ezEvade Tracker# Activator# SPred L# TS Universall everything (Minimap hack isn't working at all so I unchecked it) DZAwareness (Wich is 80% broken) EmoteSpammer

**Comment by yorik100** [comment]
PS : Was crashing before Tracker# and it was crashing before Tracker# when we was using summoner spell (Randomly)

**Comment by DrHouse12** [comment]
Yeah, for now im using HeaveStrike Ryze and its amazing ! But script not using spells in lanecleaner. Slutty Ryze has slow combo dunno why, and i tested just two scripts for now. I've got crash in random times, i will make ss from console next time and open new issue.

**Comment by djole353** [comment]
im using usually heaven strike riven elactivator sdk skin changer and tracker
